File: launchdarkly/python/src/PatchRepositoryExample.py
import json
from datetime import date, datetime
from pprint import pprint

from launchdarkly_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"ApiKey": "YOUR_API_KEY"},
)

with ApiClient(configuration) as api_client:
    patch_operation_1 = models.PatchOperation(
        op="replace",
        path="/defaultBranch",
    )

    patch_operation = [
        patch_operation_1,
    ]

    try:
        response = api.CodeReferencesApi(api_client).patch_repository(
            repo=None,
            patch_operation=patch_operation,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling CodeReferencesApi#patch_repository: %s\n" % e)

File: launchdarkly/ruby/src/PatchRepositoryExample.rb
require "json"
require "launchdarkly_client"

LaunchDarklyClient.configure do |config|
    config.api_key["ApiKey"] = "YOUR_API_KEY"
end

patch_operation_1 = LaunchDarklyClient::PatchOperation.new
patch_operation_1.op = "replace"
patch_operation_1.path = "/defaultBranch"

patch_operation = [
    patch_operation_1,
]

begin
    response = LaunchDarklyClient::CodeReferencesApi.new.patch_repository(
        nil, # repo
        patch_operation,
    )

    p response
rescue LaunchDarklyClient::ApiError => e
    puts "Exception when calling CodeReferencesApi#patch_repository: #{e}"
end
